docs/features/style-props/StyleProps.stories.mdx

```diff
@@ -26,7 +26,7 @@ To apply a style value, set a value to any [style property](#properties) of an O
     paddingBottom={3}
     marginBottom={3}
     color="alias-static-white"
-    borderRadius={3}
+    borderRadius="md"
```

Comment:
Why shifting from number to t-shirt sizes?

Does seem consistent why the other props

Reply:
With the additional "more rounded" border radius. `rounded` was not cutting it anymore, as we would need rounded and mega rounded. I saw two options, deprecate rounded and go the way of many Design System and go with t-shirt sizes or only use numbers, numbers are hard to remember and are considered global tokens.

Also rounded is implicit as a border radius is rounded, that's it's main property.

![image](https://user-images.githubusercontent.com/361632/205364455-31c6f9d4-3938-4e4e-a5f4-7fb22c05c6dc.png)

Reply:
I think I am missing something, I don't see how going from "3" to "md" makes it easier or more versatile?

Reply:
We used to have rounded, now that we need at least two radii, going back to a number does not make sense since the other are named. The main goal is constituency why having pill, circular and 1 2 3 4 (non named)? Does that makes sense?